src/go/rpk/pkg/cli/cmd/acl.go

```diff
@@ -52,13 +52,14 @@ func NewACLCommand(fs afero.Fs, mgr config.Manager) *cobra.Command {
 		configClosure,
 		&brokers,
 	)
-	tlsClosure := common.BuildTLSConfig(&certFile, &keyFile, &truststoreFile)
+	kafkaTlsClosure := common.BuildKafkaTLSConfig(&certFile, &keyFile, &truststoreFile, configClosure)
+	adminTlsClosure := common.BuildAdminApiTLSConfig(&certFile, &keyFile, &truststoreFile, configClosure)
```

Comment:
Why is this using the same certs for both the kafka tls config and the admin api tls config?

Reply:
Ohh. Good catch.

Reply:
Fixed